docker/iml-stratagem.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.39 as builder
+WORKDIR /build
+COPY . .
+RUN cd iml-services/iml-stratagem && cargo build --release
+
+FROM rust:1.39
+COPY --from=builder /build/target/release/iml-stratagem /usr/local/bin
+RUN apt-get update \
```

Comment:
I don't think we use postgres in this service.

Reply:
This one also complains:
```
iml-stratagem_1      | /usr/local/bin/wait-for-dependencies.sh: line 35: psql: command not found
```